### PR TITLE
Clean up build script, add Babel for docs js bundles

### PR DESCRIPTION
**Pull request by ndresx**
As discussed here (https://github.com/prettier/prettier/issues/1928), I have removed the no longer used `build:docs` script as well as added Babel for `index.js` and `parser-babylon.js` in the `docs` folder to make them work in browsers (like Safari) who do not support various language features - in `strict` mode - yet.

In addition, I have cleaned up the build script.

### CODE REVIEW

**Comment by ndresx** [comment]
Need to resolve the conflict - will either happen later today, or tomorrow.

File: package.json [hunk 1]
```diff
@@ -11,7 +11,9 @@
   "main": "./index.js",
   "dependencies": {},
   "devDependencies": {
+    "babel-cli": "^6.24.1",
```
Comment (vjeux) [inline package.json]: Can you use exact dependencies please

**Comment by vjeux** [comment]
By the way, as you've probably seen, the build step generates a ton of logs and warnings. I haven't had the time to try and clean them up, if that's something your interested in, that would be awesome to fix!

File: package.json (hunk 1 above)
Comment (ndresx) [inline package.json]: Have changed it.

**Comment by ndresx** [comment]
Hi, have rebased it with master now and tried to clean up the build script a little bit more, hopefully not too much when it comes to ignoring certain rollup warnings.

**Comment by vjeux** [comment]
Ohh, sorry I didn't see the update. We shouldn't update the docs mid-release since it won't reflect what you have on npm. If you could remove that, I'll merge it in. Sorry for the delay :(

**Comment by vjeux** [comment]
Actually, i'm just doing a release, so i'll just merge it in.

**Comment by ndresx** [comment]
Sorry, and thank you!

**Pull request merged**